presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -262,7 +280,22 @@ public Node visitCreateTable(SqlBaseParser.CreateTableContext context)
         if (context.COMMENT() != null) {
             comment = Optional.of(((StringLiteral) visit(context.string())).getValue());
         }
-        return new CreateTable(getLocation(context), getQualifiedName(context.qualifiedName()), visit(context.tableElement(), TableElement.class), context.EXISTS() != null, processTableProperties(context.tableProperties()), comment);
+
+        TableProperties properties;
+        if (context.tableProperties() != null) {
```

Comment:
Done. Although there are other places that repeats such null checking, not sure if I should follow.